readme/macros.md

```diff
@@ -1,6 +1,6 @@
 # Macros
```

Comment:
I made a lot of changes to this file. Just pull in the latest from main and remove the example on the bottom.